### PR TITLE
Add sponsor blog for Microsoft Azure

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @vipulgupta2048 @palnabarun

### CODE REVIEW

File: content/blog/2020-announcing-microsoft-azure-platinum-sponsor.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Title: Announcing Microsoft Azure as a Platinum Sponsor
+Date: 2020-09-23 09:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2020
+Slug: sponsor-announcement-microsoft-azure
+Summary: We are happy to introduce **Microsoft Azure** as a Platinum Sponsor for PyCon India 2020. 
+
+We are happy to introduce **Microsoft Azure** as a Platinum Sponsor for PyCon India 2020.
+
+[![Microsoft Azure](https://in.pycon.org/2020/assets/images/sponsors/<>.png)](https://azure.microsoft.com/en-in/)
```
Comment (Sc0rpi0n101) [inline content/blog/2020-announcing-microsoft-azure-platinum-sponsor.md:10]: Logo link to be added

Comment (palnabarun) [inline content/blog/2020-announcing-microsoft-azure-platinum-sponsor.md:10]:
```suggestion
[![Microsoft Azure](https://in.pycon.org/2020/assets/images/sponsors/azure.png)](https://azure.microsoft.com/en-us/?wt.mc_id=AID3022746_QSG_474804&ocid=AID3022746_QSG_474804&WT.mc_id=pyconindia-azure-cxa)
```

File: content/blog/2020-announcing-microsoft-azure-platinum-sponsor.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Microsoft is the leading platform and company for the mobile-first, cloud-first world, and its mission is to empower every person and every organization on the planet to achieve more.
+
+Microsoft has worked with Python communities to create tools and services which provide ease of development. This year at PyCon India, Microsoft is super excited to introduce two tracks at the Microsoft Learning Hub – Skilling and Community. The Skilling room is where you can skill-up your Cloud, Python, AI and Machine Learning knowledge. The Community room is the place to hear from the Community Leaders on both tech and non-tech topics.
+
+Join us at the Microsoft Learning Hub at PyCon India 2020 to learn more about Microsoft Azure.
+
+Website Link: <https://azure.microsoft.com/en-in/>
```
Comment (palnabarun) [inline content/blog/2020-announcing-microsoft-azure-platinum-sponsor.md:18]:
```suggestion
Website Link: <https://azure.microsoft.com/en-us/?wt.mc_id=AID3022746_QSG_474804&ocid=AID3022746_QSG_474804&WT.mc_id=pyconindia-azure-cxa>
```

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: content/blog/2020-announcing-microsoft-azure-platinum-sponsor.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+[![Microsoft Azure](https://in.pycon.org/2020/assets/images/sponsors/azure.png)](https://azure.microsoft.com/en-us/?wt.mc_id=AID3022746_QSG_474804&ocid=AID3022746_QSG_474804&WT.mc_id=pyconindia-azure-cxa)
+
+Microsoft is the leading platform and company for the mobile-first, cloud-first world, and its mission is to empower every person and every organization on the planet to achieve more.
+
+Microsoft has worked with Python communities to create tools and services which provide ease of development. This year at PyCon India, Microsoft is super excited to introduce two tracks at the Microsoft Learning Hub – Skilling and Community. The Skilling room is where you can skill-up your Cloud, Python, AI and Machine Learning knowledge. The Community room is the place to hear from the Community Leaders on both tech and non-tech topics.
+
+Join us at the Microsoft Learning Hub at PyCon India 2020 to learn more about Microsoft Azure.
+
+Website Link: <https://azure.microsoft.com/en-in/>
```
Comment (palnabarun) [inline content/blog/2020-announcing-microsoft-azure-platinum-sponsor.md:18]:
```suggestion
Website Link: <https://azure.microsoft.com/en-us/?wt.mc_id=AID3022746_QSG_474804&ocid=AID3022746_QSG_474804&WT.mc_id=pyconindia-azure-cxa>
```

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: content/blog/2020-announcing-microsoft-azure-platinum-sponsor.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+[![Microsoft Azure](https://in.pycon.org/2020/assets/images/sponsors/azure.png)](https://azure.microsoft.com/en-us/?wt.mc_id=AID3022746_QSG_474804&ocid=AID3022746_QSG_474804&WT.mc_id=pyconindia-azure-cxa)
+
+Microsoft is the leading platform and company for the mobile-first, cloud-first world, and its mission is to empower every person and every organization on the planet to achieve more.
+
+Microsoft has worked with Python communities to create tools and services which provide ease of development. This year at PyCon India, Microsoft is super excited to introduce two tracks at the Microsoft Learning Hub – Skilling and Community. The Skilling room is where you can skill-up your Cloud, Python, AI and Machine Learning knowledge. The Community room is the place to hear from the Community Leaders on both tech and non-tech topics.
+
+Join us at the Microsoft Learning Hub at PyCon India 2020 to learn more about Microsoft Azure.
+
+Website Link: <https://azure.microsoft.com/en-us/?wt.mc_id=AID3022746_QSG_474804&ocid=AID3022746_QSG_474804&WT.mc_id=pyconindia-azure-cxa>
```
Comment (palnabarun) [inline content/blog/2020-announcing-microsoft-azure-platinum-sponsor.md:18]:
```suggestion
Website Link: [Microsoft Azure](https://azure.microsoft.com/en-us/?wt.mc_id=AID3022746_QSG_474804&ocid=AID3022746_QSG_474804&WT.mc_id=pyconindia-azure-cxa)
```

**Review by palnabarun — CHANGES_REQUESTED** [review]

**Pull request merged**